draft-ietf-trans-rfc6962-bis.md

```diff
@@ -2048,7 +2046,7 @@ Misissued certificates that have not been publicly logged, and thus do not have
 a valid SCT, are not considered compliant. Misissued certificates that do have
 an SCT from a log will appear in that public log within the Maximum Merge Delay,
 assuming the log is operating correctly. As a log is allowed to serve an STH
-that's up to MMD old, the maximum period of time during which a misissued
+that's up to the MMD old, the maximum period of time during which a misissued
```

Comment:
Not sure the 'the' appropriate here?
My original intention was to say that the period an old STH may be served is up to this STH's age + MMD. Have not expressed it well, I believe.

Reply:
EKR wrote "Nit: up to the MMD old".

I've had another go at expressing it better.